File: 03 Berechnungen/Go/Aufgabe6.go
package main

import "fmt"

var (
	bool1 bool
	bool2 bool
)

func main() {
	bool1 = true
	bool2 = false

	fmt.Println("bool1 =", bool1, ", bool2 =", bool2)
	fmt.Println("not bool1 =", !bool1)              // nicht
	fmt.Println("bool1 && bool2 =", bool1 && bool2) // und
	fmt.Println("bool1 || bool2 =", bool1 || bool2) // oder
	fmt.Println("bool1 ^ bool2 =", bool1 != bool2)  // exklusives oder (kein richtiges xor, aber es funktioniert.)
}

File: 03 Berechnungen/Pascal/Aufgabe6.pas
Program Aufgabe6;

{$codepage utf8}

Var Bool1, Bool2: Boolean;

Begin
    Bool1 := True;
    Bool2 := False;

    WriteLn('Bool1 = ', Bool1, ', Bool2 = ', Bool2);
    WriteLn('not Bool1 = ', not Bool1);
    WriteLn('Bool1 and Bool2 = ', Bool1 and Bool2);
    WriteLn('Bool1 or Bool2 = ', Bool1 or Bool2);
    WriteLn('Bool1 xor Bool2 = ', Bool1 xor Bool2);
End.